src/Controller/EventHostsController.php

```diff
@@ -48,7 +43,7 @@ public function listHosts(Request $request, PDO $db)
             throw new Exception('Event not found', Http::NOT_FOUND);
         }
 
-        $list = $mapper->getHostsByEventId($event_id, $verbose, $start, $resultsperpage);
+        $list = $mapper->getHostsByEventId($event_id, $resultsperpage, $start, $verbose);
```

Comment:
Nice catch!

Reply:
Well, that was thanks to the type mismatch! :smile: